server/controllers/addEvent.js

```diff
@@ -2,8 +2,17 @@ const eventPost = require('../database/queries/eventPost')
 
 const addEvent = (req, res) => {
   const data = req.body
-  eventPost(data)
-  res.send()
+  const { eventTitle, eventAuthor } = data
+
+  if (!eventTitle || !eventAuthor) {
+    res
+      .status(400)
+      .json({ message: ' Fields Are Required', status: 400 })
+  } else {
+    eventPost(data)
+      .then(response => res.send(response))
+      .catch(err => res.status(500).json({ message: err.message }))
```

Comment:
careful about posting the error message into the response. The `err.message` could reveal some internal implementation details to the client - in this case, it would most likely come from the database and could give information about the DB tables and structure, which could make things easier for an attacker. You should only post generic application error messages to the client, typically for 500 errors it is just a generic `Unexpected error occurred` (or just don't send any message), for 400 errors it can be (`Missing parameter: email`), etc.